general/community/plugincontribution/adoption.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Plugins adoption programme
+sidebar_position: 9
+tags:
+  - Guidelines for contributors
+  - Plugins
+  - Plugin documentation
+---
+The Plugins adoption programme is a process for making it clear that a plugin is orphaned and is looking for a new maintainer. The programme is one of the mechanisms helping to minimise the risks of relying on additional plugins. The programme helps to find new maintainers for plugins whose original authors can't work on the plugin fully any more.
+
+The programmes started in 2014 by announcing it in [a forum post](https://moodle.org/mod/forum/discuss.php?d=260354#p1128482) and since then, many plugins found their new maintainers since then through it.
```

Comment:
```suggestion
:::info

The programmes started in 2014 by announcing it in [a forum post](https://moodle.org/mod/forum/discuss.php?d=260354#p1128482) and since then, many plugins found their new maintainers since then through it.

:::
```